rest/data/interface.go

```diff
@@ -349,4 +349,6 @@ type Connector interface {
 
 	//GetProjectSettingsEvent returns the ProjectSettingsEvents of the given identifier and ProjectRef
 	GetProjectSettingsEvent(p *model.ProjectRef) (*model.ProjectSettingsEvent, error)
+
+	CompareTasks([]string) ([]string, map[string]map[string]string, error)
```

Comment:
Just for my own understanding, the `map[string]map[string]string` is the type of a double-indexed string map with string values?

Reply:
correct